en/docs/get-started/how-to-guides/ai-code-generation.md

```diff
@@ -0,0 +1,185 @@
+# AI Integration Generation
```

Comment:
Should we rename the title for clarity? suggestion : `Generate integrations using AI`?

Reply:
Fixed with 26b89fe